OptionSelector/OSTableViewController.m

```diff
@@ -0,0 +1,157 @@
+//
+//  OSTableViewController.m
+//  OptionSelector
+//
+//  Created by Natalia Estrella on 8/9/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OSTableViewController.h"
+#import "OSDetailLIstTableViewController.h"
+#import "CQCategory.h"
+
+
+@interface OSTableViewController ()
+
+@end
+
+@implementation OSTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    CQCategory *planetCategory = [[CQCategory alloc] init];
+    
+    planetCategory.name = @"Planet";
```

Comment:
This is a good opportunity to create a separate abstract class that includes all this data, say called "ListData". This would be your "Model" in MVC. This class could have three methods that each return a CQCategory, something like planetsData, treatsData, and carsData. Then your viewDidLoad would look much small and not include the actual data:

self.cateoryObjects = @[
[ListData planetData],
[ListData treatData],
[ListData carsData]
];